test/integration/node-specific/feature_flags.test.ts

```diff
@@ -3,7 +3,7 @@ import { expect } from 'chai';
 import { MongoClient, SeverityLevel } from '../../mongodb';
 
 describe('Feature Flags', () => {
-  describe('@@mdb.skipPingOnConnect', () => {
+  describe('__skipPingOnConnect', () => {
```

Comment:
This describe block and test file name  is inaccurate now that we're not using feature flags. Can we make the naming more accurate?